### PR TITLE
upgrade package version

### PR DESCRIPTION
**Pull request by MapleWithoutWords**

### CODE REVIEW

**Comment by doddgu** [comment]
@zhenlei520 PTAL

File: Directory.Build.props [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <Project>
   <PropertyGroup>
     <MicrosoftPackageVersion>6.0.0</MicrosoftPackageVersion>
-    <MasaUtilsPackageVersion>0.5.0</MasaUtilsPackageVersion>
-    <MasaContribPackageVersion>0.5.0</MasaContribPackageVersion>
-    <MasaBlazorPackageVersion>0.5.0</MasaBlazorPackageVersion>
+    <MasaUtilsPackageVersion>1.0.0-rc.1</MasaUtilsPackageVersion>
```
Comment (zhenlei520) [inline Directory.Build.props]: It is recommended to use the same parameter to control the MASA Framework version

File: src/Api/Masa.EShop.Api.Open/Callers/Ordering/OrderingCaller.cs [hunk 2]
```diff
@@ -7,29 +7,28 @@ public class OrderingCaller : HttpClientCallerBase
...
-    private string prefix = "/api/v1/orders/";
 
     public OrderingCaller(
         IServiceProvider serviceProvider,
         IOptions<Settings> settings,
         ILogger<OrderingCaller> logger) : base(serviceProvider)
     {
         BaseAddress = settings.Value.OrderingUrl;
-        Name = nameof(OrderingCaller);
+        Prefix = "/api/v1/orders/";
```
Comment (zhenlei520) [inline src/Api/Masa.EShop.Api.Open/Callers/Ordering/OrderingCaller.cs]: It is more recommended to use the overload to change the prefix

```csharp 
protected override string Prefix { get; set; } = "/api/v1/orders/";
```

File: src/Services/Masa.EShop.Services.Catalog/Program.cs [hunk 3]
```diff
@@ -19,11 +19,13 @@
...
     .AddScoped<ICatalogTypeRepository, CatalogTypeRepository>()
     .AddScoped<ICatalogBrandRepository, CatalogBrandRepository>()
-    .AddDaprEventBus<IntegrationEventLogService>(options =>
+    .AddMasaDbContext<CatalogDbContext>(dbOptions => dbOptions.UseSqlServer("server=masa.eshop.services.eshop.database;uid=sa;pwd=P@ssw0rd;database=catalog"))
+    .AddIntegrationEventBus<IntegrationEventLogService>(options =>
     {
-        options.UseEventLog<CatalogDbContext>()
+        options.UseDapr()
+               .UseEventLog<CatalogDbContext>()
                .UseEventBus()
```
Comment (zhenlei520) [inline src/Services/Masa.EShop.Services.Catalog/Program.cs]: Delete line:8, and modify

```csharp
.UseEventBus(eventBuilder => eventBuilder.UseMiddleware(typeof(ValidatorMiddleware<>))
```

> They have the same effect and are only recommended

File: src/Services/Masa.EShop.Services.Catalog/_Imports.cs [hunk 4]
```diff
@@ -35,3 +35,5 @@
 global using System.IO.Compression;
 global using System.Linq.Expressions;
 global using System.Text.RegularExpressions;
+global using Masa.BuildingBlocks.Data.UoW;
```
Comment (zhenlei520) [inline src/Services/Masa.EShop.Services.Catalog/_Imports.cs]: Use global sort

File: src/Services/Masa.EShop.Services.Payment/_Imports.cs [hunk 5]
```diff
@@ -5,15 +5,17 @@
 global using Masa.BuildingBlocks.Ddd.Domain.Entities.Full;
 global using Masa.BuildingBlocks.Ddd.Domain.Events;
 global using Masa.BuildingBlocks.Ddd.Domain.Repositories;
+global using Masa.Contrib.Ddd.Domain.Repository.EFCore;
```
Comment (zhenlei520) [inline src/Services/Masa.EShop.Services.Payment/_Imports.cs:9]: Use global using sort

**Review by zhenlei520 — CHANGES_REQUESTED** [review]

File: src/Services/Masa.EShop.Services.Catalog/Program.cs [hunk 6]
```diff
@@ -24,7 +23,7 @@
     {
         options.UseDapr()
                .UseEventLog<CatalogDbContext>()
-               .UseEventBus()
+               .UseEventBus(eventBusBuilder=>eventBusBuilder.UseMiddleware(typeof(ValidatorMiddleware<>)))
```
Comment (zhenlei520) [inline src/Services/Masa.EShop.Services.Catalog/Program.cs]: format code

**Review by zhenlei520 — CHANGES_REQUESTED** [review]

**Review by zhenlei520 — APPROVED** [review]
LGTM @MayueCif

**Review by MayueCif — APPROVED** [review]

**Pull request merged**